enterprise/config/buildbuddy.local.yaml

```diff
@@ -13,8 +13,58 @@ storage:
     root_directory: /tmp/buildbuddy_enterprise
   enable_chunked_event_logs: true
 cache:
+  # Sample flow if migration disk -> pebble
```

Comment:
I really liked reading this example config with your comments to get an idea of how this would be used

Reply:
Glad it was helpful!